e2e-tests/portal/cypress/e2e/journeys/read-only/deal/cannot-modify-an-unissued-deal.spec.js

```diff
@@ -12,7 +12,7 @@ context('A read-only role viewing a bond that can be issued', () => {
     bonds: [],
   };
 
-  before(() => {
+  beforeEach(() => {
```

Comment:
Was this change deliberate (even though we delete the deals in an `after` and not an `afterEach`)?

Reply:
Ah added to before each too -- so it does nothing persistent to the test db